mocks/groundPlaces.ts

```diff
@@ -0,0 +1,40 @@
+const fakeGroundPlacesJSON = JSON.stringify({
```

Comment:
Why do you stringify ? Seems not necessary :)

Reply:
I may be wrong but to be able to use a JSON  in TS, you have to pass it as a string, right? 

I use JSON.stringify here to be able to use a JSON fake in my tests, how else would you do it ?

Reply:
Using a JSON still need to stringify this file
![Capture d’écran 2020-11-16 à 16 13 15](https://user-images.githubusercontent.com/19270653/99270101-d0177a80-2826-11eb-92a7-24a49497148c.png)

Reply:
did you try `import * as myFakeJson from 'path/to/jsonFile.json` ?

Reply:
Yep, same problem :(